src/ui/util.rs

```diff
@@ -14,6 +14,19 @@ pub fn get_search_results_highlight_state(
     )
 }
 
+pub fn get_artist_highlight_state(app: &App, block_to_match: ArtistBlock) -> (bool, bool) {
+    let current_route = app.get_current_route();
+    if let Some(artist) = &app.artist {
+        (
+            artist.artist_selected_block == block_to_match,
+            current_route.hovered_block == ActiveBlock::ArtistBlock
```

Comment:
For ease of reading this in the future, would you be able to assign these conditions to variables? e.g.

```rust 
let is_hovered = artist.artist_selected_block == block_to_match;
let is_selected = current_route.hovered_block == ActiveBlock::ArtistBlock && artist.artist_hovered_block == block_to_match;

(is_hovered, is_selected)
```

Reply:
I modified relevant part as you suggested.